platform/migration/platform-migration/src/main/java/org/codice/ddf/configuration/migration/ConfigurationMigrationManagerMBean.java

```diff
@@ -34,4 +35,11 @@
      * @throws MigrationException thrown if one or more Configurations couldn't be exported
      */
     Collection<MigrationWarning> export(@NotNull String exportDirectory) throws MigrationException;
+
+    /**
+     * Get a collection of the migratables available for export
+     *
+     * @return Collection of type Describable
+     */
+    Collection<Describable> getInfo();
```

Comment:
This is going to be interesting. I'm not sure how Jolokia will handle the collection of `Describable` objects. We may have to create a `DescribableImpl` class and create an instance from each `DataMigratable` we have in our list.

If we need to go down that path, we should consider having a separate class for the MBean implementation instead of having all the code in the `ConfigurationMigrationManager` class like we do right now. That'd be cleaner.